tests/e2e/tests/pilot/externalservice_test.go

```diff
@@ -58,37 +66,150 @@ func TestServiceEntry(t *testing.T) {
+			url:               "https://www.google.co.in/",
 			shouldBeReachable: true,
 		},
 		{
-			name:              "REACHABLE_de.wikipedia.org_over_wikipedia_wildcard",
-			config:            "testdata/networking/v1alpha3/wildcard-tls-wikipedia.yaml",
-			url:               "https://de.wikipedia.org/wiki/Wikipedia:Hauptseite",
+			name:              "REACHABLE_uk.google.com_no_vs_over_google_wildcard",
+			config:            "testdata/networking/v1alpha3/wildcard-https-google-no-vs.yaml",
+			url:               "https://www.google.co.uk/",
 			shouldBeReachable: true,
 		},
 		{
-			name:              "UNREACHABLE_www.wikipedia.org_over_wikipedia_wildcard",
-			config:            "testdata/networking/v1alpha3/wildcard-tls-wikipedia.yaml",
+			name:              "UNREACHABLE_in.google.com_no_vs_over_google_wildcard_80",
+			config:            "testdata/networking/v1alpha3/wildcard-https-google-no-vs.yaml",
+			url:               "http://www.google.co.in/",
+			shouldBeReachable: false,
+		},
+		{
+			name:              "UNREACHABLE_uk.google.com_no_vs_over_google_wildcard_80",
+			config:            "testdata/networking/v1alpha3/wildcard-https-google-no-vs.yaml",
+			url:               "http://www.google.co.uk/",
+			shouldBeReachable: false,
+		},
+		{
+			name:              "UNREACHABLE_google.com_no_vs_over_google_wildcard_",
+			config:            "testdata/networking/v1alpha3/wildcard-https-google-no-vs.yaml",
+			url:               "https://www.google.com",
+			shouldBeReachable: false,
+		},
+		{
+			name:              "UNREACHABLE_google.com_no_vs_over_google_wildcard_",
+			config:            "testdata/networking/v1alpha3/wildcard-https-google-no-vs.yaml",
+			url:               "http://www.google.org",
+			shouldBeReachable: false,
+		},
+		{
+			name:              "REACHABLE_google_no_vs_over_multihosts_wildcard",
+			config:            "testdata/networking/v1alpha3/wildcard-https-multihosts-no-vs.yaml",
+			url:               "https://www.google.com",
+			shouldBeReachable: true,
+		},
+		{
+			name:              "REACHABLE_bing_no_vs_over_multihosts_wildcard",
+			config:            "testdata/networking/v1alpha3/wildcard-https-multihosts-no-vs.yaml",
+			url:               "https://www.bing.com",
+			shouldBeReachable: true,
+		},
+		{
+			name:              "UNREACHABLE_google_no_vs_over_multihosts_wildcard",
+			config:            "testdata/networking/v1alpha3/wildcard-https-multihosts-no-vs.yaml",
+			url:               "http://www.google.com",
+			shouldBeReachable: false,
+		},
+		{
+			name:              "UNREACHABLE_bing_no_vs_over_multihosts_wildcard",
+			config:            "testdata/networking/v1alpha3/wildcard-https-multihosts-no-vs.yaml",
+			url:               "http://www.bing.com",
+			shouldBeReachable: false,
+		},
+		{
+			name:              "UNREACHABLE_wikipedia_no_vs_over_multihosts_wildcard",
+			config:            "testdata/networking/v1alpha3/wildcard-https-multihosts-no-vs.yaml",
```

Comment:
all these are redundant with the yahoo test. can be removed

Reply:
which yahoo test? :). I decided to use www.google.co.uk and www.google.co.in as they work fine.